docs/modding/suitmodding/index.md

```diff
@@ -14,7 +14,7 @@ We recommend that you start at the beginning and work your way up, as the early
 This is a list of tools that are essential for any kind of Hitman modding. The tutorials will assume you have them installed and will ask you to make frequent use of them.
 
 -   [RPKG Tool](https://glaciermodding.org/rpkg/)
--   [Simple Mod Framework](https://www.nexusmods.com/hitman3/mods/200)
+-   [Simple Mod Framework](https://github.com/atampy25/simple-mod-framework/releases/latest)
```

Comment:
Talk about how you need to also set the runtime path in QNE to the game's runtime path. Otherwise, Load from Game will not work (as per your parts tutorial).

Reply:
I had that in the first tutorial but yes, prudent to have in requirements. Done